sources/osg/Matrix.js

```diff
@@ -1170,8 +1341,94 @@ define( [
             };
         } )(),
 
-        // compute the 4 corners vector of the frustrum
-        computeFrustrumCornersVectors: function ( projectionMatrix, vectorsArray ) {
+        clampProjectionMatrix: function ( projection, znear, zfar, nearFarRatio, resultNearFar ) {
```

Comment:
In shadowMap, as a simple projection matrix clamp:
https://github.com/cedricpinson/osgjs/blob/feature/multifrustum/sources/osgShadow/ShadowMap.js#L701
(comparing with what I get using the optimisation, could be that we can letting it as an alternative ShadowFrustumAlgo )